src/DynamoCoreWpf/UI/DefaultBrandingResourceProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using Dynamo.Wpf.Interfaces;
+
+namespace Dynamo.Wpf.UI
+{
+    class DefaultBrandingResourceProvider : IBrandingResourceProvider
+    {
+        public ImageSource GetImageSource(ResourceName resourceName)
+        {
+            ImageSource image = null;
+            switch (resourceName)
+            {
+                case ResourceName.AboutBoxLogo:
+                    image = new BitmapImage(
+                    new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/AboutWindow/logo_about.png",
+                        UriKind.Absolute));
+                    break;
+            }
+            return image;
+        }
+
+        public string GetStringResource(ResourceName resourceName)
```

Comment:
To align with the other API method `GetImageSource`, this method should have the form of `GetXxxYyy` where `XxxYyy` is the type this method returns (without the word `Resource` since that is implied by the interface name "Resource Provider"). Please rename this method and I'll merge it right in!